src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1060,7 +1069,7 @@ internal ValueTask<IBackendMessage> ReadMessage(bool async, DataRowLoadingMode d
                         // TODO: There could be room for optimization here, rather than the async call(s)
                         ReadBuffer.Timeout = TimeSpan.FromMilliseconds(InternalCommandTimeout);
                         for (; _pendingPrependedResponses > 0; _pendingPrependedResponses--)
-                            await ReadMessageLong(DataRowLoadingMode.Skip, false, true, cancellationToken2);
+                            await ReadMessageLong(DataRowLoadingMode.Skip, false, true, false, cancellationToken2);
```

Comment:
nit: we should add named parameters here, 3 bool literals in a row isn't very clear